packages/hooks/src/hooks.ts

```diff
@@ -68,3 +68,23 @@ export function useElementNode<ElementType = HTMLElement>() {
 
   return [element, elementRefCallback] as const;
 }
+
+export function useDocumentEventListener(
+  enabled: boolean,
```

Comment:
Do we need to make this a required prop? I'd think this would default to `true`.

Reply:
+1

Reply:
Done!